### ISSUE TITLE
Include the test class in the Description

### ISSUE THREAD

**Issue by dimo414** [square/burst#61]
I'm trying to design a JUnit4 Rule and have run into an issue with tests using Burst, because the `Description` passed to my rule returns null from `getTestClass()`.

It looks like [this line](https://github.com/square/burst/blob/dbe2dad/burst-junit4/src/main/java/com/squareup/burst/BurstRunner.java#L45) is the source of the `Description` in question, as it passes a custom string instead of a [`className`](https://junit.org/junit4/javadoc/4.12/org/junit/runner/Description.html#createTestDescription(java.lang.String,%20java.lang.String,%20java.lang.annotation.Annotation...)).

Would Burst be open to fixing this? Something like:

```java
return Description.createTestDescription(
    getTestClass().getJavaClass(),
    getName() + ":" + method.getName(),
    method.getAnnotations());
```

Is sufficient for my use case, though it does change the result of `Description.getDisplayName()`. If that works I'm happy to send a PR, otherwise are there other approaches you might be open to?

**Comment by JakeWharton** [comment]
I think this is probably fine, but it's been so long the nuances of descriptions are definitely lost on me. The only real concern, I think, is the effect on the IDE experience.

**Comment by dimo414** [comment]
Yeah, I'm not sure if there's a specific format for the second argument that would be most-correct, but seeing as nothing's blown up with the malformed `className` I'd hope IDEs can handle a change like this? :crossed_fingers:

**Comment by saket** [comment]
Turns out this bug is also restricting [paparazzi](https://github.com/cashapp/paparazzi) from working with burst.
